lib/modules/manager/nuget/extract.ts

```diff
@@ -51,7 +51,7 @@ function extractDepsFromXml(xmlNode: XmlDocument): NugetPackageDependency[] {
       const dep = getDep(child.val, true);
 
       if (is.nonEmptyStringAndNotWhitespace(dep.depName)) {
-        results.push({ ...dep, depType: 'docker' });
+        results.push({ ...dep, depName: dep.depName!, depType: 'docker' });
```

Comment:
```suggestion
        results.push({ ...dep, depName: dep.depName, depType: 'docker' });
```

Ok, typescript doesn't detect that `...dep` has `depName` as defined but `depName: dep.depName. 😕